### ISSUE TITLE
Accessibility: The names of the parameters are not read by the screen reader only the values

### ISSUE THREAD

**Issue by Lo-lo78** [JoepVanlier/ysfx#110]
**Is your feature request related to a problem? Please describe.**

YSFX is largely very accessible but when I move among the parameters I can only read the value but not the name.
So if I have many parameters it cannot help my memory.

**Describe the solution you'd like**

I wish that when I move the name of the parameter and not just the value with the tab key with the parameters.

**Describe alternatives you've considered**

Yes it is precisely the name of the slider who is not read.:
"Oscillator Volume: 1.0"
With the Screen Reader I can only read 1.0 but not the name of the parameter.

**Additional context**

**Comment by Lo-lo78** [comment]
To make experiments you can try NVDA.
It is one screen reader free and open source:
https://www.nvaccess.org/
